core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2253,6 +2268,12 @@ public String toString() {
     public static final String WORKER_WEB_PORT = "alluxio.worker.web.port";
     public static final String WORKER_UFS_BLOCK_OPEN_TIMEOUT_MS =
         "alluxio.worker.ufs.block.open.timeout";
+    public static final String WORKER_UFS_INSTREAM_CACHE_EXPIRE_MS =
+        "alluxio.worker.ufs.instream.cache.expire.ms";
```

Comment:
how about calling this `alluxio.worker.ufs.instream.cache.expiration.time`? we no more require the time-based conf property to suffix with `ms`

Reply:
sgtm